types/src/proof/unit_tests/proof_test.rs

```diff
@@ -328,128 +324,6 @@ fn test_verify_transaction() {
     assert!(proof.verify(&ledger_info, 1).is_err());
 }
 
-#[test]
-fn test_verify_state_store_resource_and_event() {
```

Comment:
state proof should still be tested? cc @lightmark

Reply:
This test case doesn't test anything other than event proof :)